spec/controllers/tasks_controller_spec.rb

```diff
@@ -79,6 +78,13 @@
         expect(task.reload).not_to be_completed
       end
     end
+
+    context "when the task has a notify method" do
+      it "calls the notify method" do
```

Comment:
The comment says `notify` but the method tested is `send_emails`. Let's fix this. Or this might just go away if [my suggestion above](https://github.com/Tahi-project/tahi/pull/750/files#r22566201) is implemented.